src/models/encounter/useEncounterFieldSchemas.js

```diff
@@ -61,6 +57,11 @@ export default function useSightingFieldSchemas() {
         };
       });
 
+      speciesOptions.push({
+        label: intl.formatMessage({ id: 'UNKNOWN' }),
```

Comment:
Not clear to me why this should be hard-coded as unknown.

Reply:
it's just an option in the species dropdown to remove the existing species. so for example if they accidentally put zebra they can switch it back to unknown to remove the species property